src/ActiveLogin.Authentication.BankId.AspNetCore/Events/BankIdCollectPendingEvent.cs

```diff
@@ -0,0 +1,22 @@
+using ActiveLogin.Authentication.BankId.Api.Models;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore.Events
+{
+    /// <summary>
+    /// Event for collect pending authentication order.
+    /// </summary>
+    public class BankIdCollectPendingEvent : BankIdEvent
+    {
+        public BankIdCollectPendingEvent(string orderRef, CollectHintCode hintCode)
+            : base(BankIdEventTypeIds.BankIdCollectPending, BankIdEventTypeNames.BankIdCollectPending, EventSeverity.Information)
+        {
+            OrderRef = orderRef;
+            HintCode = hintCode;
+        }
+
+        public string OrderRef { get; set; }
```

Comment:
All properties on the events could be made private set. They should not be changed once set.

Reply:
Changed all properties in every event to have only the default readonly setter.